### PR TITLE
[model zoo test] check result

### PR DESCRIPTION
**Pull request by xuhongyao**
use FileCheck to check result

### CODE REVIEW

File: models/vision/classification/alexnet/run_alexnet.sh [hunk 1]
```diff
@@ -16,8 +17,15 @@ $curr_dir/../../get_images.sh $image_dir
 if [[ $TEST_WITH_GPU -eq 1 ]]; then
   echo "======== Testing with ODLA TensorRT ========"
   python3 $curr_dir/../../invoke_halo.py --model $model_file --label-file $curr_dir/../1000_labels.txt --image-dir $image_dir --odla tensorrt
+# RUN: FileCheck --input-file /tmp/alexnet_tensorrt.txt %s
```
Comment (weimingzha0) [inline models/vision/classification/alexnet/run_alexnet.sh:20]: how is "/tmp/alexnet_tensorrt.txt" generated?
should we use something like "RUN: %s | FileCheck %s" ?

File: models/vision/classification/alexnet/run_alexnet.sh [hunk 2]
```diff
@@ -16,8 +17,15 @@ $curr_dir/../../get_images.sh $image_dir
...
 fi
 
 # Using HALO to compile and run inference with ODLA XNNPACK
 echo "======== Testing with ODLA DNNL ========"
-python3 $curr_dir/../../invoke_halo.py --model $model_file --label-file $curr_dir/../1000_labels.txt --image-dir $image_dir --odla dnnl
\ No newline at end of file
+python3 $curr_dir/../../invoke_halo.py --model $model_file --label-file $curr_dir/../1000_labels.txt --image-dir $image_dir --odla dnnl
+# RUN: FileCheck --input-file /tmp/alexnet_dnnl.txt %s
+
+# CHECK: /tmp/images/dog.jpg ==> "wallaby, brush kangaroo",
```
Comment (weimingzha0) [inline models/vision/classification/alexnet/run_alexnet.sh:28]: Can we do "CHECK: dog.jpg ==> "...." 

Matching for "/tmp/images" might cause issues if the temporary dir name is changed in future.

File: models/vision/classification/alexnet/run_alexnet.sh (hunk 1 above)
Comment (xuhongyao) [reply models/vision/classification/alexnet/run_alexnet.sh:20]: "/tmp/alexnet_tensorrt.txt" is generated by  invoke_halo.py line 196-205.
FileCheck not work well for shell & python script's screen output

File: models/vision/classification/alexnet/run_alexnet.sh (hunk 2 above)
Comment (xuhongyao) [reply models/vision/classification/alexnet/run_alexnet.sh:28]: yes, you are right

File: models/vision/classification/alexnet/run_alexnet.sh (hunk 1 above)
Comment (weimingzha0) [reply models/vision/classification/alexnet/run_alexnet.sh:20]: I see

**Review by youbeny — APPROVED** [review]

**Review by weimingzha0 — APPROVED** [review]

**Pull request merged**